app/views/npq_separation/admin/delivery_partners/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1 class="govuk-heading-l">Delivery partners</h1>
+
+<%= govuk_button_link_to "Add a delivery partner", new_npq_separation_admin_delivery_partner_path %>
+
+<%=
+  govuk_table do |table|
+    table.with_head do |header|
+      header.with_row do |row|
+        row.with_cell(text: "Delivery partner")
```

Comment:
I think we want to expose the ECF ID here - will help lead provider queries for the contracts team